### ISSUE TITLE
The overlay does not show at all

### ISSUE THREAD

**Issue by Silejonu** [trigg/Discover#126]
I an unable to see the overlay at all.
I am on Arch Linux, and I use GNOME on Wayland (I tested with X11 and it didn't work either). Discord is installed via Flatpak (I tested the native Arch package, which didn't work either), and I use the `discover-overlay` package in the AUR. I installed `python-gobject` and `gtk-layer-shell`.

I am under the impression I'm not missing anything, but I still don't get anything. Is there anything I can do?

Here is what I get when launching `discover-overlay` from the terminal:
```
** (discover-overlay:5614): WARNING **: 21:09:51.287: It appears your Wayland compositor does not support the Layer Shell protocol

** (discover-overlay:5614): CRITICAL **: 21:09:51.287: layer_surface_new: assertion 'gtk_wayland_get_layer_shell_global ()' failed

** (discover-overlay:5614): WARNING **: 21:09:51.287: Falling back to XDG shell instead of Layer Shell (surface should appear but layer features will not work)

** (discover-overlay:5614): CRITICAL **: 21:09:51.287: Custom wayland shell surface is not a layer surface, your Wayland compositor may not support Layer Shell

** (discover-overlay:5614): CRITICAL **: 21:09:51.287: Custom wayland shell surface is not a layer surface, your Wayland compositor may not support Layer Shell

** (discover-overlay:5614): CRITICAL **: 21:09:51.287: Custom wayland shell surface is not a layer surface, your Wayland compositor may not support Layer Shell

** (discover-overlay:5614): CRITICAL **: 21:09:51.287: Custom wayland shell surface is not a layer surface, your Wayland compositor may not support Layer Shell

** (discover-overlay:5614): CRITICAL **: 21:09:51.287: Custom wayland shell surface is not a layer surface, your Wayland compositor may not support Layer Shell

** (discover-overlay:5614): CRITICAL **: 21:09:51.290: layer_surface_new: assertion 'gtk_wayland_get_layer_shell_global ()' failed

** (discover-overlay:5614): WARNING **: 21:09:51.290: Falling back to XDG shell instead of Layer Shell (surface should appear but layer features will not work)

** (discover-overlay:5614): CRITICAL **: 21:09:51.290: Custom wayland shell surface is not a layer surface, your Wayland compositor may not support Layer Shell

** (discover-overlay:5614): CRITICAL **: 21:09:51.290: Custom wayland shell surface is not a layer surface, your Wayland compositor may not support Layer Shell

** (discover-overlay:5614): CRITICAL **: 21:09:51.290: Custom wayland shell surface is not a layer surface, your Wayland compositor may not support Layer Shell

** (discover-overlay:5614): CRITICAL **: 21:09:51.290: Custom wayland shell surface is not a layer surface, your Wayland compositor may not support Layer Shell

** (discover-overlay:5614): CRITICAL **: 21:09:51.290: Custom wayland shell surface is not a layer surface, your Wayland compositor may not support Layer Shell
INFO:root:Could not find monitor : None
INFO:root:Could not find monitor : None
INFO:root:Could not find monitor : None
INFO:root:Loading saved channel 0
INFO:root:Could not find monitor : None
INFO:root:Could not find monitor : None
INFO:root:Loading saved channel 0
INFO:root:Could not find monitor : None
INFO:root:Could not find monitor : None
INFO:root:Could not find monitor : None
INFO:root:Autostart info : desktop /usr/share/applications/discover_overlay.desktop auto None
WARNING:root:Getting 222 rooms
```

**Comment by trigg** [comment]
Gnome on wayland absolutely won't work until they support GTK layer shell.

As for gnome on X11 I'm unsure as to why it wouldn't show up there. @midget3111  yours still working?

**Comment by Silejonu** [comment]
My bad, I tested again and it currently works fine on X11.
I had tested it a few months back on both X11 and Wayland, and I couldn't get it to work on either of those at the time. When I tried to troubleshoot again those past few days, I tried only Wayland, and since I was getting the same error messages as before, I did not bother to check with X11 again. I definitely should have taken the time to do so, sorry.
I guess I'll have to switch to X11 when I want to have the overlay, for now, then.
Sorry for the erroneous bug report, I'm closing it.
